credentials-generator/helpers.js

```diff
@@ -33,20 +32,14 @@ const cloneJSON = data => JSON.parse(JSON.stringify(data, null, 2));
  *
  * @returns {Promise<object>} - Returns the resulting did key driver result.
  */
-const getDiDKey = async ({seedMultiBase = _seed} = {}) => {
+const getDiDKey = async ({
```

Comment:
```suggestion
const getDidKey = async ({
```

`DiD` is incorrect (it doesn't follow a consistent naming pattern)... It's either `getDidKey` or `getDIDKey`... and I think we prefer the former.